packages/cozy-harvest-lib/src/services/budget-insight.js

```diff
@@ -86,7 +87,7 @@ const createTemporaryToken = async ({ client, konnector, account }) => {
     client,
     jobResponse.data.attributes,
     'result',
-    30 * 1000
+    TEMP_TOKEN_TIMOUT_S * 1000
```

Comment:
> We saw that when BI is long to respond, it takes usually 35s to 40s but
in most cases it takes 2s to 3s.

Why not increasing to 3s instead?

Reply:
Dans la plupart des cas, la durée est de 2 à 3 secondes. Mais dans certains cas assez rares (mais difficiles à dénombrer en totalité car plusieurs problèmes provoquent le même symptôme) cela dure plus de 30s et cela abouti à une erreur fatale pour l'utilisateur.

Cela arrive suffisamment pour que des utilisateurs le remontent régulièrement.

Et je préfère mettre limite à 60s au lieu de 33 ou 34 pour avoir de la marge